packages/kit/test/utils.js

```diff
@@ -181,7 +186,13 @@ export const test = base.extend({
 /** @type {import('@playwright/test').PlaywrightTestConfig} */
 export const config = {
 	// generous timeouts on CI, especially on windows
-	timeout: process.env.CI ? (process.platform === 'win32' ? 45000 : 30000) : 10000,
+	timeout: process.env.CI
```

Comment:
can you add some parentheses here or will prettier remove them? I'm having a hard time reading this

Reply:
or maybe a better solution would be to make this always be something like `60000 / num_cpus`

Reply:
of maybe we don't need to get so fancy with this and can just set a single constant if we stop creating more workers than CPUs

Reply:
Yeah, prettier removed them...  Let me put in the simpler logic I had in the PR comment.